libs/application/core/src/types/ApplicationTypes.ts

```diff
@@ -67,4 +68,8 @@ export const ApplicationConfigurations = {
     slug: 'samstarf',
     translation: 'ia.application',
   },
+  [ApplicationTypes.APPLICATION_FOR_FUNDING_GOVERNMENT_PROJECTS]: {
+    slug: 'umsokn-um-fjarmognun-rikisverkefni',
```

Comment:
Same here, the url is already containing `umsoknir/` as base URL it would make more sense to have only `umsoknir/fjarmognun-rikisverkefni` right?

Reply:
good point ill change it 👍